client/tracing/src/block/mod.rs

```diff
@@ -142,16 +138,10 @@ impl Subscriber for BlockSubscriber {
 		}
 	}
 
-	fn record_follows_from(&self, _span: &Id, _follows: &Id) {
-		// Not currently used
-		unimplemented!("record_follows_from is not implemented");
-	}
-
 	fn event(&self, event: &tracing::Event<'_>) {
 		let mut values = crate::Values::default();
 		event.record(&mut values);
-		let parent_id = event.parent().cloned()
-			.or_else(|| self.current_span.id());
+		let parent_id = event.parent().cloned();
```

Comment:
I assume this is `None` if no parent?

Reply:
Yes, it's ok for the root span to miss the `parent_id`?

Reply:
Yes